src/Tasks/Microsoft.Common.CurrentVersion.targets

```diff
@@ -1516,17 +1516,26 @@ Copyright (C) Microsoft Corporation. All rights reserved.
     ====================================================================================
                                         _GetProjectReferenceTargetFrameworkProperties
 
-    Builds the GetTargetFrameworkProperties target of all existent project references,
-    passing $(TargetFrameworkMoniker) as $(ReferringTargetFramework) and sets the
-    SetTargetFramework metadata of the project reference to the value that is returned.
-
-    This allows a cross-targeting project to select how it should be configured to
-    build against the most appropriate target for the referring target framework.
+    Builds the GetTargetFrameworks target of all existent project references to get a list
+    of all supported TargetFrameworks of the referenced projects. For each project, call
+    GetReferenceNearestTargetFrameworkTask to determine the closest match. This allows a
```

Comment:
Reword this to say "Call GetReferenceNearestTargetFrameworkTask to determine the closest match for each project."  This way it won't imply that the task is called once for project.